<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>545</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>759</width>
    <height>545</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>759</width>
    <height>545</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>517</width>
      <height>517</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="blueSlider">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="redSlider">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="greenSlider">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="redSpin">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>52</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="greenSpin">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>160</y>
      <width>52</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="redSpin_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>52</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QWidget" name="displayWidget" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>blueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>715</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>718</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>656</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redSpin_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>717</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSpin_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>717</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>657</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>713</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>656</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
